src/desktop/components/view_in_room/view.coffee

```diff
@@ -166,6 +164,6 @@ module.exports = class ViewInRoom extends Backbone.View
           @$img.css visibility: 'visible'
         if @$sourceImage
           @$sourceImage.css visibility: 'visible'
-        @scrollbar.reenable()
+        document.body.style.overflowY = 'visible'
```

Comment:
Nice! We recently fixed an [unexpected behavior](https://github.com/artsy/reaction/pull/2074) of setting `overflow` to `auto`, instead of `visible`!